tests/common/snappi/common_helpers.py

```diff
@@ -436,68 +444,112 @@ def config_wred(host_ans, kmin, kmax, pmax, profile=None):
             return False
 
         """ Ensure that Kmin is no larger than Kmax during the update """
-        if kmin > kmin_old:
-            host_ans.shell('sudo ecnconfig -p {} -gmax {}'.format(p, kmax))
-            host_ans.shell('sudo ecnconfig -p {} -gmin {}'.format(p, kmin))
 
+        gmx_cmd = 'sudo ecnconfig -p {} -gmax {}'
+        gmin_cmd = 'sudo ecnconfig -p {} -gmin {}'
+
+        if asic_value != 'None':
+            gmx_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmax {}' % asic_value
+            gmin_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmin {}' % asic_value
+            if asic_type == 'broadcom':
```

Comment:
Since we are pausing the packets in the queue when we send pause frames, the DNX will flush the queues every 500ms. To stop this , we are disabling the credit watchdog timer

Reply:
> Since we are pausing the packets in the queue when we send pause frames, the DNX will flush the queues every 500ms. To stop this , we are disabling the credit watchdog timer

This seems to be similar to packet aging feature in some devices. Can you search "disable_packet_aging" and see how we use it?